SDL_UDP_Networking/SDL_net_test/User.h

```diff
@@ -48,8 +48,8 @@ class User
 		level = _level;
 	}
 
-	void print() {
-		std::cout << "Username : " << username << "Level : " << level << "\n";
+	std::string to_string() {
+		return "Username : " + username + "Level : " + std::to_string(level);
```

Comment:
Put all your raw text in constants in one location so you don't have to go hunting through files to update them.